src/documentation/print-dataflow-graph-wiki.ts

```diff
@@ -1056,7 +1056,7 @@ Let's start by looking at the properties of the dataflow information object: ${O
 
 ${ (() => {
 			/* this includes the meta field for timing */
-			guard(Object.keys(result).length === 8, () => 'Update Dataflow Documentation!'); return ''; 
+			guard(Object.keys(result).length === 9, () => 'Update Dataflow Documentation!'); return '';
```

Comment:
this should indicate that we add at least a sentence explaining why the quickCfg may be there :D